vue/src/views/CropsView.vue

```diff
@@ -1,31 +1,33 @@
-<!-- <template>
-  <header-bar title="Crops"></header-bar>
-  <div class="p-6"></div>
-</template> -->
-
 <template>
-  <!-- <div class="px-3 py-5 text-h2 text-primary_hover">
-    Crops
-  </div> -->
-  <div class="ml-28">
-    <header-bar
-      title="Crops"
-      :widgetOptions="widgetOptionsCrops"
-      :storeOptions="storeOptions"
-    ></header-bar>
-    <div class="row p-6">
-      <div v-if="storeOptions.indexOf('crops-table') > -1" class="col-8">
-        <crops-table title="Overview" :visibleColumns="columns"></crops-table>
-      </div>
-      <div v-if="storeOptions.indexOf('crops-map') > -1" class="col-4 pl-2">
-        <crops-map></crops-map>
-      </div>
+  <header-bar
+    title="Crops"
+    :widgetOptions="widgetOptionsCrops"
+    :storeOptions="storeOptions"
+  ></header-bar>
+  <div class="row p-6">
+    <div v-if="storeOptions.indexOf('crops-table') > -1" class="col-8">
+      <crops-table
+        ref="cropsTableRef"
+        title="Overview"
+        :crops="crops"
+        :visibleColumns="columns"
+        @row-enter="tableCropsEnter"
+        @row-leave="tableCropsLeave"
+      ></crops-table>
+    </div>
+    <div v-if="storeOptions.indexOf('crops-map') > -1" class="col-4 pl-2">
+      <crops-map
+        ref="cropsMapRef"
+        :crops="crops"
+        @polygon-enter="mapCropsEnter"
+        @polygon-leave="mapCropsLeave"
+      ></crops-map>
     </div>
   </div>
 </template>
 
 <script setup lang="ts">
-import { onMounted, type Ref } from "vue";
+import { onMounted, type Ref, ref } from "vue";
```

Comment:
removed unused import
```suggestion
import { type Ref, ref } from "vue";
```